frontend/src/static/js/v_zoom.js

```diff
@@ -1,17 +1,21 @@
 /* global Vuex */
 
+function initialState() {
```

Comment:
Can you provide some link/document for the rationale to convince us that it is a more elegant way of `resetting some default attribute` comparing with our previous methods?

Reply:
It is the first time for me to see such implementation.

Reply:
Oh is there a previous method you used? I just copied this off Stackoverflow as a nice way to avoid rewriting the entire initial state since I need to reset to the initial state in some method.

Reply:
Got it